tests/SonataPageAdminTraitTest.php

```diff
@@ -0,0 +1,705 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the behat/helpers project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Ekino\BehatHelpers;
+
+use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Element\DocumentElement;
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Behat\Mink\Session;
+use Behat\Mink\WebAssert;
+use Ekino\BehatHelpers\SonataPageAdminTrait;
+use phpmock\Mock;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author William JEHANNE <william.jehanne@ekino.com>
+ */
+class SonataPageAdminTraitTest extends TestCase
```

Comment:
can you write a setUp method to write some properties? it would avoid same lines again and again at the beginning of each test